test/xpu_api/numerics/complex.number/complex.transcendentals/acosh.pass.cpp

```diff
@@ -29,7 +29,7 @@ template <class T>
 void
 test()
 {
-    test(dpl::complex<T>(INFINITY, 1), dpl::complex<T>(INFINITY, 0));
+    test(dpl::complex<T>(TestUtils::infinity_val, 1), dpl::complex<T>(TestUtils::infinity_val, 0));
```

Comment:
So, this is a nit pick, and probably not worth caring about, because I think all standard library implementations we see have a float infinity, but technically the standard doesn't require it
https://en.cppreference.com/w/cpp/types/numeric_limits/has_infinity

Just leaving this comment to note that, but I don't think anything needs to be done.  If we wanted this to be portable to any possible STL implementation, we should probably check `has_infinity` for the types at compile time before continuing with any INFINITY tests.